Artesian/Artesian.SDK/Factory/AuctionTimeSerie.cs

```diff
@@ -129,9 +129,10 @@ private AddAuctionTimeSerieOperationResult _add(LocalDateTime bidTime, AuctionBi
         /// <param name="deferCommandExecution">DeferCommandExecution</param>
         /// <param name="deferDataGeneration">DeferDataGeneration</param>
         /// <param name="keepNulls">if <see langword="false"/> nulls are ignored (server-side). That is the default behaviour.</param>
+        /// <param name="upsertMode">Upsert Mode</param>
         /// <param name="ctk">The Cancellation Token</param> 
         /// <returns></returns>
-        public async Task Save(Instant downloadedAt, bool deferCommandExecution = false, bool deferDataGeneration = true, bool keepNulls = false, CancellationToken ctk = default)
+        public async Task Save(Instant downloadedAt, bool deferCommandExecution = false, bool deferDataGeneration = true, bool keepNulls = false, UpsertMode? upsertMode = null, CancellationToken ctk = default)
```

Comment:
This is a Major library change as old code would no longer compile if caller used positional arguments like

x.Save(now, false, false, false, ctk)

provide overloads to demote to Minor release.

Major release requires Migration Guide

Reply:
I could be trying to implement the change incorrectly. 

If I add the overload in IMarketData,
```
Task Save(Instant downloadedAt, bool deferCommandExecution = false, bool deferDataGeneration = true, bool keepNulls = false, CancellationToken ctk = default)
Task Save(Instant downloadedAt, bool deferCommandExecution = false, bool deferDataGeneration = true, bool keepNulls = false, UpsertMode? upsertMode = null, CancellationToken ctk = default)
```
Then the call, x.Save(instant) becomes ambiguous between the two versions of the method and the code doesn't build. Is there a way to define a default method in this case?

Reply:
I've added overloads for save. But, not sure if this is the correct. I'm going to discuss with Michele and Niccolo so see if we want to go down the breaking route with optional parameters.